fp.sql

```diff
@@ -70,17 +82,86 @@ gather_contact_data AS (
     GROUP BY
         site_short
 ),
-join_data AS (
+--For KPI % of students admitted to Best-Fit College choose to enroll at Best-Fit College
+gather_best_fit_data AS (
+    SELECT
+        contact_id,
+        site_short,
+        --Students accepted to Best Fit
+        (
+            SELECT
+                student_c
+            FROM
+                `data-warehouse-289815.salesforce_clean.college_application_clean` AS subq1
+            WHERE
+                (
+                    admission_status_c = "Accepted"
+                    AND College_Fit_Type_Applied_c = "Best Fit"
+                )
+                AND Contact_Id = student_c
+            group by
+                student_c
+        ) AS accepted_best_fit,
+        --Students enrolled in Best Fit
+        (
+            SELECT
+                student_c
+            FROM
+                `data-warehouse-289815.salesforce_clean.college_application_clean` AS subq2
+            WHERE
+                admission_status_c IN ("Accepted and Enrolled", "Accepted and Deferred")
+                AND fit_type_enrolled_c = "Best Fit"
+                AND Contact_Id = student_c
+            group by
+                student_c
+        ) AS accepted_enrolled_best_fit
+    FROM
+        `data-warehouse-289815.salesforce_clean.contact_template`
+    WHERE
+        college_track_status_c = '11A'
+        AND (
+            grade_c = "12th Grade"
+            OR (
+                grade_c = 'Year 1'
+                AND indicator_years_since_hs_graduation_c = 0
+            )
+        )
+),
+--Define numerator and denominator for KPI % of students admitted to Best-Fit College enroll at Best-Fit College
+prep_best_fit_enrollment_kpi AS (
     SELECT
         site_short,
+        SUM(
+            CASE
+                WHEN accepted_best_fit IS NOT NULL THEN 1
+                ELSE 0
+            END
+        ) AS fp_accepted_best_fit_numerator,
```

Comment:
@isabelreyna  I might be reading this backwards, but it looks like your numerator and denominator is switched?

I would think the denominator should be all students who were accepted, and the numerator is all students who were accepted and then enrolled. 

But it looks like you have it the opposite way?

Reply:
@jbakerr  You're right. I labeled them backwards. Will update

Reply:
@jbakerr Change made